README.md

```diff
@@ -15,6 +15,44 @@ The Playbook documents how the Playbook gets changed. See the
 [This Playbook](https://playbook.dxw.com/#this-playbook) section for more
 information.
 
+## Development guide
+
+### Dependencies
+
+These dependencies aren't installed by a setup script as people have their own
```

Comment:
I do wonder if we should be more prescriptive about this. If people dont use rbenv they are on their own for instance.

Reply:
Have updated this after our discussion on Slack, as we tend to install `rbenv` etc here.